GitUI/OsShellUtil.cs

```diff
@@ -18,7 +18,7 @@ public static void OpenAs(string file)
 
         public static void SelectPathInFileExplorer(string filePath)
         {
-            Process.Start("explorer.exe", "/select, " + filePath);
+            Process.Start("explorer.exe", "/select, \"" + filePath + "\"");
```

Comment:
We have an extension method
```suggestion
            Process.Start("explorer.exe", $"/select, {filePath.Quote()}");
```